src/Components/Web.JS/src/Rendering/BrowserRenderer.ts

```diff
@@ -423,7 +431,13 @@ function countDescendantFrames(batch: RenderBatch, frame: RenderTreeFrame): numb
   }
 }
 
-function raiseEvent(event: Event, browserRendererId: number, eventHandlerId: number, eventArgs: EventForDotNet<UIEventArgs>, eventFieldInfo: EventFieldInfo | null) {
+function raiseEvent(
+  event: Event,
+  browserRendererId: number,
+  eventHandlerId: number,
+  eventArgs: EventForDotNet<UIEventArgs>,
+  eventFieldInfo: EventFieldInfo | null
+): Promise<void> {
```

Comment:
AFAICT, nothing uses the returned promise. Event dispatch is currently fire-and-forget.

To make this clearer in the usage patterns, could we define this method as returning `void` and likewise have `setEventDispatcher` take a function that returns `void`?

We can always make it accept a promise in the future if some need arises.

Reply:
made everything void returning